backend/api/views/main.py

```diff
@@ -74,6 +52,26 @@ def create_person():
     )
 
 
+# function that is called when you visit /census_response
+@main.route("/response_rates", methods=["GET"])
+def get_response_rates():
+    responses_rate = None
+
+    if "date" in request.args:
+        date = request.args["date"]
+        if "state" in request.args:
+            response_rates = get_response_rates_by_state(request.args["state"], date)
+        else:
+            response_rates = get_response_rates_by_date(date)
+
+    elif "year" in request.args:
+        response_rates = get_response_rates_by_year(request.args["year"])
+    else:
+        return create_response(status=442, message="Missing request parameters")
+
+    return create_response(data={"response_rates": response_rates})
```

Comment:
We should also document the specific combinations of parameters that we're supporting